tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/DocumentType/DocumentType.spec.ts

```diff
@@ -0,0 +1,765 @@
+    // Arrange
+    await umbracoUi.documentType.goToSection(ConstantHelper.sections.settings);
+
+    // Act
+    await umbracoUi.documentType.clickActionsMenuAtRoot();
+    await umbracoUi.documentType.clickCreateButton();
+    await umbracoUi.documentType.clickCreateDocumentTypeButton();
+    await umbracoUi.documentType.enterDocumentTypeName(documentTypeName);
+    await umbracoUi.documentType.clickSaveButton();
+
+    // Assert
+    await umbracoUi.documentType.isSuccessNotificationVisible();
+    expect(await umbracoApi.documentType.doesNameExist(documentTypeName)).toBeTruthy();
+    await umbracoUi.documentType.reloadTree('Document Types');
+    await umbracoUi.documentType.isDocumentTreeItemVisible(documentTypeName);
+  });
+
+  test('can create a document type with a template @smoke', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    await umbracoUi.documentType.goToSection(ConstantHelper.sections.settings);
+    await umbracoApi.template.ensureNameNotExists(documentTypeName);
+
+    // Act
+    await umbracoUi.documentType.clickActionsMenuAtRoot();
+    await umbracoUi.documentType.clickCreateButton();
+    await umbracoUi.documentType.clickCreateDocumentTypeWithTemplateButton();
+    await umbracoUi.documentType.enterDocumentTypeName(documentTypeName);
+    await umbracoUi.documentType.clickSaveButton();
+
+    // Assert
+    // Checks if both the success notification for document Types and teh template are visible
+    await umbracoUi.documentType.doesSuccessNotificationsHaveCount(2);
+    // Checks if the documentType contains the template
+    const documentTypeData = await umbracoApi.documentType.getByName(documentTypeName);
+    const templateData = await umbracoApi.template.getByName(documentTypeName);
+    expect(documentTypeData.allowedTemplates[0].id).toEqual(templateData.id);
+    expect(await umbracoApi.documentType.doesNameExist(documentTypeName)).toBeTruthy();
+
+    // Clean
+    await umbracoApi.template.ensureNameNotExists(documentTypeName);
+  });
+
+  test('can create a element type @smoke', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    await umbracoUi.documentType.goToSection(ConstantHelper.sections.settings);
+
+    // Act
+    await umbracoUi.documentType.clickActionsMenuAtRoot();
+    await umbracoUi.documentType.clickCreateButton();
+    await umbracoUi.documentType.clickCreateElementTypeButton();
+    await umbracoUi.documentType.enterDocumentTypeName(documentTypeName);
+    await umbracoUi.documentType.clickSaveButton();
+
+    // Assert
+    await umbracoUi.documentType.isSuccessNotificationVisible();
+    expect(await umbracoApi.documentType.doesNameExist(documentTypeName)).toBeTruthy();
+    // Checks if the isElement is true
+    const documentTypeData = await umbracoApi.documentType.getByName(documentTypeName);
+    expect(documentTypeData.isElement).toBeTruthy();
+  });
+
+  test.describe('Design tab', () => {
```

Comment:
To avoid the long tests file, The tests for the Design tab, Structure tab, and Template tab should be moved into separate files

Reply:
The tests for the Design tab, Structure tab, and Template tab still be included in Document Type tests class.

Reply:
Since there are nearly 40 tests in the "DocumentType.spec.ts file. I would suggest to separate it into 4 individual files: one file for general tests for Document Type, one for Design tab, one for Structure tab and one for Template tab. It will make our code easier to manage and maintain.